lib/formats/html/html2form.flow

```diff
@@ -608,12 +608,13 @@ getHtmlPictureMagnifyWithCloseButton(
 		);
 
 		cursorB = make(DefaultCursor());
+		isNotScrollB = make(true);
 		form =
 			Interactive(
 				[
-					MouseClick(nop1),
+					MouseClick(\__ -> nextDistinct(isNotScrollB, true)),
```

Comment:
MouseClick is called on 'mouseup', so it shouldn't help. MouseDown would be better.

Reply:
we realized that MouseClick and MouseDown are the same